src/components/footer/footer.jsx

```diff
@@ -2,15 +2,37 @@ import React from 'react';
 
 import './footer.scss';
 
-import logo from '../../images/nouri-logo.png'
+import logo from '../../images/nouri-logo.png';
 
-class Footer extends React.Component {
+// getting Font Awesome import to component
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFacebook, faMedium, faTwitter, faLinkedin } from '@fortawesome/free-brands-svg-icons';
+
+const facebookSocial = <FontAwesomeIcon icon={ faFacebook } />
+const linkedinSocial = <FontAwesomeIcon icon={ faLinkedin } />
+const mediumSocial = <FontAwesomeIcon icon={ faMedium } />
+const twitterSocial = <FontAwesomeIcon icon={ faTwitter } />
 
+class Footer extends React.Component {
   render() {
-    return <footer>
+    return (
+      <footer>
       <img className="footer-logo" src={logo} />
+
+      <div className="social-media">
+        <p>Follow us on Social Media</p>
+    
+          <div className="social-icons">
+            <a href="https://linkedin.com/company/teamnouri">{linkedinSocial}</a>
```

Comment:
Read up on a tags and how to optimize them for accessibility and SEO.

Reply:
In terms of accessibility, you're rendering these links with SVGs as their "content". For somebody that can't see the symbols - they'll have no context of what they're being linked to. FontAwesome icons don't use a `title` element to provide context about their icon so it's up to you to provide text that a screen reader would parse.

I recommend creating a ScreenReaderOnly component that has invisible, but rendered text. Your seeing-eye users will see the icons, and your screen reader-using users will be given the proper context. A good example would be:

```
<a href="https://linkedin.com/company/teamnouri">
  <ScreenReaderOnly>teamnouri on LinkedIn</ScreenReaderOnly>
  {linkedinSocial}
</a>
```

This methodology will also help you earn points with SEO audits for having descriptive text in hyperlinks.

See: https://github.com/OperationCode/front-end/tree/master/components/ScreenReaderOnly

Reply:
@kylemh thank you for the specific guidance Kyle. I appreciate it. I am working to incorporate those changes. @ambergkim Thank you.